lib/fluent/plugin/out_azure_servicebus_queue.rb

```diff
@@ -58,5 +71,21 @@ def generateToken(url,key_name,key_value)
 
       "SharedAccessSignature sr=#{target_uri}&sig=#{signature}&se=#{expires}&skn=#{key_name}"
     end
+
+    # reference1: https://github.com/microsoft/fluent-plugin-azure-storage-append-blob/blob/master/lib/fluent/plugin/out_azure-storage-append-blob.rb
```

Comment:
Can you add some explanation behind the reference links? I don't really know what to look for in the reference or how it relates to the block of code below

Reply:
added.